pkgs/applications/misc/octoprint/default.nix

```diff
@@ -2,7 +2,8 @@
 , stdenv
 , lib
 , fetchFromGitHub
-, python3
+, python38
```

Comment:
Why not override python3 to python38 in top-level?

Reply:
Tracking the python environment (interpreter & overrides) in two different files doesn't seem like a great idea.